common/document_parser/lib/responsibility_parse/responsibility_parser.py

```diff
@@ -118,7 +119,8 @@ def format_responsibility_results(self, resp_list, file_name, title):
                     "organizationPersonnelEntities": ";".join(resp_intro_entities),
                     "responsibilityNumbering": "",
                     "responsibilityText": "",
-                    "responsibilityEntities": ""
+                    "responsibilityEntities": "",
+                    "status": "active"
```

Comment:
I think the unit tests need to be updated with `status` 
<img width="1021" alt="Screen Shot 2022-10-26 at 11 23 07 AM" src="https://user-images.githubusercontent.com/105378552/198068004-067d7de5-8286-43da-b50e-5e26a89bb952.png">